src/sentry/static/sentry/app/views/releasesV2/detail/overview/index.tsx

```diff
@@ -82,19 +83,28 @@ class ReleaseOverview extends AsyncView<Props> {
               {({crashFreeTimeBreakdown, ...releaseStatsProps}) => (
                 <ContentBox>
                   <Main>
-                    {hasHealthData && (
+                    {hasHealthData ? (
```

Comment:
Does `hasHealthData` imply user has Discover feature?

Reply:
Not necessarily, we are still discussing this. I added a TODO comment just to be sure we won't forget about it 👍